eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
```

Comment:
I'd probably put this service account in the clusterloader2 namespace.

Reply:
It's just much easier to clean up by running a kubectl delete namespace clusterloader2